### ISSUE TITLE
Add support for websocket client

### ISSUE THREAD

**Issue by zdila** [Huachao/vscode-restclient#173]
It would be really cool for the extension to also support websockets. If it is out of scope then it may serve as an idea for new extension ;-).

**Comment by Huachao** [comment]
@zdila It's a bit of out of scope, I'd like to put it in backlog. Thanks for your suggestion.

**Comment by zydxhs** [comment]
It would be really cool for the extension to also support websockets.
Hopefully it will be supported in a later release.

**Comment by jeffreyschultz** [comment]
I would also like to vote for websocket support in this extension, or the creation of another. I really like how I can drop an .http file into the project right alongside the controller and provide quick access to related rest calls. That would be amazing for websockets as well.

**Comment by mepsoid** [comment]
I'd like to vote up for websockets support too.

**Comment by alersrt** [comment]
Agreed with previous comments. Developing with websockets is not such rare case. From the other side it can be complicate enough to design it for proper using. So for example I don't see the proper way to implement it and use in http files restclient was designed for from the start.

**Comment by PolyphonyRequiem** [comment]
I'd love websocket support as well

**Comment by tkachukcha** [comment]
I'd like support of websockets too!

**Comment by haxung** [comment]
I'd like support of websockets too!

**Comment by vikas0605** [comment]
+1, that would be really cool feature indeed.

**Comment by aneasystone** [comment]
So, are there any plans to support this feature?

**Comment by csc-jm** [comment]
After refactoring the API of my project to utilize websockets, the `.http` files are useless now. Plz add websocket support :(

**Comment by DmitriiKholkin** [comment]
+1 for supporting websockets!

**Comment by suhli** [comment]
+1 for supporting websockets!
